SPECS/kernel/config

```diff
@@ -510,6 +510,10 @@ CONFIG_CPU_IBPB_ENTRY=y
 CONFIG_CPU_IBRS_ENTRY=y
 CONFIG_CPU_SRSO=y
 # CONFIG_GDS_FORCE_MITIGATION is not set
+CONFIG_MITIGATION_RFDS=y
+CONFIG_SPECTRE_BHI_ON=y
```

Comment:
New BHI options introduced here: https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/f825494f2c6fab421c5c59b5def321775c825818
Default option changed to ON here: https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/8d8dc7ee5b98fca7a65c800df93ded071a96fd49